ci/glci/util.py

```diff
@@ -361,14 +362,19 @@ def release_set_manifest_name(
     version: str,
     flavourset_name: str,
     build_type: glci.model.BuildType,
+    with_timestamp: bool = False,
 ):
     BT = glci.model.BuildType
 
     if build_type in (BT.SNAPSHOT, BT.DAILY):
-        return f'{gardenlinux_epoch}-{build_committish[:6]}-{flavourset_name}'
+        name = f'{gardenlinux_epoch}-{build_committish[:6]}-{flavourset_name}'
     elif build_type is BT.RELEASE:
-        return f'{version}-{flavourset_name}'
+        name = f'{version}-{flavourset_name}'
```

Comment:
in general, I recommend adding a `else: raise NotImplementeError(value)` to have an easier way of identifying all relevant code-paths when extending the enum w/ additional values

Reply:
__

Reply:
Done.